lua/lazyvim/plugins/extras/coding/codecompanion.lua

```diff
@@ -0,0 +1,47 @@
+return {
+  {
+    "olimorris/codecompanion.nvim",
+    cmd = { "CodeCompanion", "CodeCompanionActions", "CodeCompanionToggle", "CodeCompanionAdd", "CodeCompanionChat" },
+
+    opts = function()
+      local options = require("codecompanion.config")
+      local user = vim.env.USER or "User"
+
+      options.strategies.chat.roles = {
+        llm = "  CodeCompanion",
+        user = "  " .. user:sub(1, 1):upper() .. user:sub(2),
+      }
+
+      options.strategies.chat.keymaps.close.modes = {
+        n = "q",
+        i = "<C-c>",
+      }
+      options.strategies.chat.keymaps.stop.modes.n = "<C-c>"
+      options.strategies.chat.keymaps.save.modes.n = "gS"
```

Comment:
`save` is no longer a valid entry in keymaps; I believe it was renamed to `send`

Reply:
tks for the heads up!!
I also ended changing save.modes.n to <CR>, I think it may be better for LazyVim users, but it's totally an opinion.